src/toil/test/batchSystems/batchSystemTest.py

```diff
@@ -237,6 +236,29 @@ def testSetEnv(self):
             self.assertEqual(jobUpdateInfo.exitStatus, 23)
             self.assertEqual(jobUpdateInfo.jobID, job5)
 
+        def test_set_job_env(self):
+            """ Test the mechanism for setting per-job environment variables to batch system jobs."""
+            script = 'if [ "x${FOO}" == "xbar" ] ; then exit 23 ; else exit 42 ; fi'
+            command = "bash -c \"\\${@}\" bash eval " + script.replace(';', r'\;')
+
+            # Issue a job with a job environment variable
+            jobDesc6 = self._mockJobDescription(command=command, jobName='test6', unitName=None,
```

Comment:
Shouldn't the variables in here be `snake_case`?

Reply:
My bad... that was copied from the previous test.